src/FPIndia/Jobs.hs

```diff
@@ -14,13 +14,85 @@ import System.UnionMount qualified as UM
 import UnliftIO
 
 data Job = Job
-  { jobName :: !Text
-  , jobLocation :: !Text
+  { jobName :: !String
+  , jobSource :: !SourceType
+  , jobWebsite :: !String
+  , --  , jobWebsite :: !SourceUrl
+    jobLocation :: !Text
+  , jobLanguages :: !Language
```

Comment:
How can this field represent more than one language? In `jobs.csv` you have `[Haskell | Purescript]`.

Reply:
You can use `NonEmpty Language` (or `[Language]` if zero languages is a possibility)

Reply:
I think we should keep the possiblity of empty languages right now